payjoin-cli/Cargo.toml

```diff
@@ -2,7 +2,10 @@
 name = "payjoin-cli"
 version = "0.0.1"
 authors = ["Dan Gould <d@ngould.dev>", "Martin Habovstiak <martin.habovstiak@gmail.com>"]
-edition = "2018"
+repository = "https://github.com/payjoin/rust-payjoin"
+readme = "README.md"
+edition = "2021"
+resolver = "2"
```

Comment:
@jbesraa May you please share why this is necessary to introduce this change? I'm not so familiar with this part of the build process

Reply:
resolver = "2" is a flag to add latest deps resolution features. But I actually dont think we need it because we have edition = "2021" and in this edition the default resolver is =2.

https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2